coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4226,27 +4242,32 @@ def gelu(context, node):
     context.add(res)
 
 
-@register_torch_op(torch_alias=["slice"])
-def _slice(context, node):
-    inputs = _get_inputs(context, node, expected=5)
+@register_torch_op(torch_alias=["slice_copy.tensor"])
```

Comment:
We probably want to at least keep `_slice` (with the leading underscore) as an alias, right?

Reply:
I guess `_slice` has been deprecated? I searched [PyTorch op definition](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/native_functions.yaml) but didn't find any `_slice`, so I removed it

Reply:
Make sense, there is no downside. Let's keep it then. Added to an alias as suggested